elftools/dwarf/descriptions.py

```diff
@@ -312,12 +317,29 @@ def _describe_attr_block(attr, die, section_offset):
     DW_LANG_UPC: '(Unified Parallel C)',
     DW_LANG_D: '(D)',
     DW_LANG_Python: '(Python)',
+    DW_LANG_OpenCL: '(OpenCL)', # Not tested
```

Comment:
What does "not tested" mean here?
AFAICT most of the entries above were not tested

Reply:
I've added all the language codes that are listed in DWARF5. However, I didn't match the externally visible language designations (as seen in readelf data dumps) against the sources of GNU readelf. Since by now I've got the binutils' sources anyway, I will check that.

Reply:
Thanks!

Reply:
Matched against binutils, a couple of minor fixes.